mx-semantics/main/accounts/tools.md

```diff
@@ -0,0 +1,152 @@
+```k
+
+module MX-ACCOUNTS-TOOLS
+    imports private BOOL
+    imports private COMMON-K-CELL
+    imports private INT
+    imports private MX-ACCOUNTS-CONFIGURATION
+    imports private MX-COMMON-SYNTAX
+    imports private STRING
+    imports private MX-ACCOUNTS-STACK-CONFIGURATION
+
+    syntax MxInstructions ::= transferESDT
+                                  ( source: String
+                                  , destination: String
+                                  , token: String
+                                  , nonce: Int
+                                  , value: Int
+                                  )
+                            | checkAccountExists(address: String)
+                            | checkESDTBalance(account: String, token: String, value: Int)
+                            | addToESDTBalance(account: String, token: String, delta: Int, allowNew: Bool)
```

Comment:
Do we intend to have negative deltas to `addToESDTBalance`? If it is only intended to add amounts to the total balance, then perhaps we should have `requires` in the rules preventing modifications by negative values and possibly negative balances. Or perhaps change the naming from `addToESDTBalance` to something like `modifyESDTBalanceByDelta`.

Reply:
Yes, the deltas can be negative, see https://github.com/Pi-Squared-Inc/rust-demo-semantics/pull/70/files#diff-39d9f39de40db7c63b5cb4ada9dfd2b5c889ffadbae02c26a7db378356ef234cR50-R51. I copied the name from the existing mx-semantics. I renamed it, is it better now?

Reply:
It is, @virgil-serbanuta . Thank you!